src/app/main/main.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-main',
+  templateUrl: './main.component.html',
+  styleUrls: ['./main.component.sass']
+})
+export class MainComponent implements OnInit {
+  main = true;
+  localWrapper = false;
+  userCard = JSON.parse(localStorage.getItem('user'));
+  userName = '';
+  userEmail = '';
+  userPhone = '';
+  userImage = {};
+  randomNumber = '';
+  constructor(private router: Router) {
+
+  }
+  goToInformationPage() {
+    this.router.navigate(['/information']);
+  }
+  ngOnInit(): void {
+    if (localStorage.length !== 0) {
```

Comment:
move work with localstorage in service
realize methods like are `get, set, remove, check`

Reply:
I prefer using DI token for this, but this was not clearly described in task. We can ignore this case of using LocalStorage for the sake of simplicity.